### ISSUE TITLE
build_vmprovision retirement default settings

### ISSUE THREAD

**Issue by jeffmcutter** [rhtconsulting/miq-BASE#3]
Automatic retirement dates by default are potentially hazardous.  Someone utilizing this code might unknowingly leave auto retirement in place leading to a situation where VM's are unintentionally retired post implementation.  Is it possible we might change the default behavior to not set any retirement dates by default?  Perhaps we can put a value in the instance to enable the setting of retirement dates?  I would be willing to help with modifying the code if there is agreement.

Thanks,
-Jeff

**Comment by jsimonelli** [comment]
@ramrexx what are your thoughts?

**Comment by ramrexx** [comment]
I agree they are dangerous. In our lab we were asked to have mandatory retirement dates so that our lab would continually clean itself up. Why not just leave the code in there and comment out the call to the method?

**Comment by jeffmcutter** [comment]
I would say, that would address the concern as long as it's commented out by default.  My thought for adding an attribute to the instance is to provide visibility to the possibility to enable automatic retirement there, but is not necessary to alleviate this risk.

Thanks,
-Jeff
